src/pages/LogView/LoadingPage/index.tsx

```diff
@@ -37,6 +39,12 @@ const LoadingPage: React.FC<LoadingPageProps> = ({ logType }) => {
   let url = "";
   let htmlLogURL = "";
   let jobLogsURL = "";
+  const { data: logkeeperMetadata } = useFetch<LogKeeperMetadata>(
+    getResmokeLogURL(buildID || "", { testID, metadata: true }),
```

Comment:
do we need the `|| "" `?
```suggestion
    getResmokeLogURL(buildID, { testID, metadata: true }),
```

Reply:
We do since `buildID` is not always defined. We use the skip to avoid making the query with the broken url but we still need to pass in an empty string to appease the type guards in `getResmokeLogURL`